scripts/extensions/sams/src/interfaces.ts

```diff
@@ -27,6 +27,7 @@ export interface ISetItem extends IBaseRestApiResponse {
     destination_name?: string;
     destination_config?: Dictionary<string, any>;
     destination?: IStorageDestinationItem;
+    count?: number;
```

Comment:
It would make it easier for a developer to understand resources from the API if the `I*Item` interfaces use the same schema as defined in the server (this could also cause issues when attempting to save these items, i.e. which attributes are for the browser and which ones are for the API).

In this case it would be better if the Set Asset counts were to be stored in the `Set` branch of the Redux store.
In this case you would need to:
In `store/sets/types.ts`
* Add a `Dictionary<string, number>` to `ISetState`
* Add a new action type to store the set counts

In `store/sets/actions.ts`
* A new thunk action that will call the `api.assets.getCount()`

In `store/sets/reducers.ts`
* Add a new `case` for storing these set counts into the state

In `store/sets/selectors.ts`
* Add a selector for these set counts.

Once the above is done, you should then be able to use the selector in `SetListItem`